js/views/pauseScreen.js

```diff
@@ -147,43 +130,50 @@ class PauseScreen extends React.Component {
     }
   }
 
-  handlePlayerMouseDown(e) {
+  /**
+   * call handlePlayerMouseDown when mouseDown was called on document and videoType is Vr
+   * @param {Event} event - mouse down event object
+   */
+  handlePlayerMouseDown(event) {
     if (this.props.controller.videoVr) {
-      e.persist();
+      event.persist();
     }
     this.props.controller.state.accessibilityControlsEnabled = true;
     this.props.controller.state.isClickedOutside = false;
-    this.props.handleVrPlayerMouseDown(e);
+    this.props.handleVrPlayerMouseDown(event);
   }
 
-  handlePlayerMouseMove(e) {
-    this.props.handleVrPlayerMouseMove(e);
+  /**
+   * call handlePlayerMouseMove when mouseMove was called on document and videoType is Vr
+   * @param {Event} event - mouse move event object
+   */
+  handlePlayerMouseMove(event) {
+    this.props.handleVrPlayerMouseMove(event);
   }
 
   /**
    * call handleVrMouseUp when mouseup was called on selectedScreen
-   * @param {Event} e - event object
+   * @param {Event} event - event object
    */
-  handlePlayerMouseUp(e) {
-    e.stopPropagation(); // W3C
-    e.cancelBubble = true; // IE
+  handlePlayerMouseUp(event) {
+    event.stopPropagation(); // W3C
+    event.cancelBubble = true; // IE
   }
 
   /**
    * call handleVrMouseUp when mouseup was called on document
-   * @param {Event} e - event object
+   * @param {Event} event - event object
    */
-  handleVrMouseUp(e) {
-    this.props.handleVrPlayerMouseUp(e);
+  handleVrMouseUp(event) {
```

Comment:
The same as for `handleVrTouchEnd`. Not required wrapper.

Reply:
\+

Reply:
\+

Reply:
Was this updated?

Reply:
oh, I missed out it, thank you!

\+